src/moveFile.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable no-console */
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const moveFile = (oldPath, newPath) => {
+  const destPath = fs.existsSync(newPath)
+    ? path.join(newPath, path.basename(oldPath))
+    : newPath;
+
+  const isDestDir = destPath.endsWith('/');
+
+  const destDir = isDestDir ? destPath.slice(0, -1) : path.dirname(destPath);
```

Comment:
The slicing operation to remove the trailing slash is not necessary. path.dirname works fine with trailing slashes.